app/views/schools/choose_programme/show.html.erb

```diff
@@ -9,39 +9,39 @@
         <span class="govuk-caption-xl"><%= @school.name %></span>
 
         <%= form_for @induction_choice_form, url: schools_choose_programme_path, method: :post do |f| %>
-        <%= f.govuk_error_summary %>
+            <%= f.govuk_error_summary %>
             <%= f.govuk_collection_radio_buttons(
-                :programme_choice,
-                @induction_choice_form.programme_choices,
-                :id,
-                :name,
-                legend: { text: "How do you want to run your training in #{@induction_choice_form.cohort.start_year} to #{@induction_choice_form.cohort.start_year + 1}?", size: "xl", tag: "h1" }
+              :programme_choice,
+              @induction_choice_form.programme_choices,
+              :id,
+              :name,
+              legend: { text: "How do you want to run your training in #{@induction_choice_form.cohort.start_year} to #{@induction_choice_form.cohort.start_year + 1}?", size: "xl", tag: "h1" }
             ) do %>
-
-                <p class="govuk-body">If you’re not sure which option to choose,
+              <p class="govuk-body">If you’re not sure which option to choose,
                 <%= govuk_link_to "check our guidance on statutory changes and new training programmes (opens in new tab)",
-                    "https://www.gov.uk/government/publications/early-career-framework-reforms-overview/early-career-framework-reforms-overview",
-                    target: :_blank,
-                    rel: "noopener noreferrer",
-                    class: "govuk-link govuk-link--no-visited-state" %>
-                </p>
-                <p class="govuk-body">You can also contact your
+                                  "https://www.gov.uk/government/publications/early-career-framework-reforms-overview/early-career-framework-reforms-overview",
+                                  target: :_blank,
+                                  rel: "noopener noreferrer",
+                                  class: "govuk-link govuk-link--no-visited-state" %>
```

Comment:
Not really important here but you can pass `no_visited_state: true` to `govuk_link_to` for the same effect.

Reply:
Great thanks. Much less verbose!.

Reply:
After Christmas it will support `new_tab: true` which means you'd be able to drop ` target: :_blank, rel: "noopener noreferrer"` too, but I [need to wait for Ruby 2.7 support to drop before merging](https://github.com/DFE-Digital/govuk-components/pull/363).